KlingonAssistant/data/mem-12-q.xml

```diff
@@ -1303,14 +1305,14 @@ The subtitle was "What's she doing?" but a more literal translation is "Why is s
       <column name="entry_name">qaw</column>
       <column name="part_of_speech">n</column>
       <column name="definition">memory (ability)</column>
-      <column name="definition_de"></column>
+      <column name="definition_de">Gedächtnis, Erinnerungsvermögen</column>
       <column name="synonyms"></column>
       <column name="antonyms"></column>
       <column name="see_also">{qaw:v}</column>
       <column name="notes">This refers to "memory" in the sense of the ability to remember or the power of recall. It does not refer to specific memories or recollections, for which see {wov'on:n}.
 
 Someone who has an excellent memory may be said to have a {qaw pov:n@@qaw:n, pov:v}.</column>
-      <column name="notes_de"></column>
+      <column name="notes_de">Wenn jemand ein gutes Gedächtnis hat, kann man sagen, er habe ein {qaw pov:n@@qaw:n, pov:v}</column>
```

Comment:
This is only the last sentence. Did you mean to leave the others out?

Reply:
Yes, indeed. The German definition is not ambiguous as the english word, so there is no need to distinguish.